app/controllers/environments_controller.rb

```diff
@@ -45,20 +45,23 @@ def edit
 
   # POST /environments
   def create
+    user = check_user(params[:user_id].to_i, 'You are not allowed to create an environment for this user.')
```

Comment:
You can't add any new required fields to this, params[:user_id] cannot be assumed to be present.

Reply:
I guess if you want further clarification, it is because Jupyter Classic and Jupyter Lab extensions use this same exact endpoint